wpilibj/src/athena/java/edu/wpi/first/wpilibj/SensorBase.java

```diff
@@ -89,10 +89,10 @@ public static void setDefaultSolenoidModule(final int moduleNumber) {
   protected static void checkSolenoidModule(final int moduleNumber) {
     if (!SolenoidJNI.checkSolenoidModule(moduleNumber)) {
       StringBuilder buf = new StringBuilder();
-      buf.append("Requested solenoid module is out of range. Minimumm: 0, Maximum: ");
-      buf.append(kPCMModules);
-      buf.append(", Requested: ");
-      buf.append(moduleNumber);
+      buf.append("Requested solenoid module is out of range. Minimumm: 0, Maximum: ")
+        .append(kPCMModules)
+        .append(", Requested: ")
+        .append(moduleNumber);
```

Comment:
The rule violation claims that this is slightly faster. 

>Consecutively calls to StringBuffer/StringBuilder .append should reuse the target object. This can improve the performance.

http://pmd.sourceforge.net/pmd-5.3.2/pmd-java/rules/java/strings.html